merlin/models/tf/transforms/tensor.py

```diff
@@ -101,6 +101,13 @@ def call(self, inputs: TabularData, **kwargs) -> TabularData:
             elif isinstance(val, tf.RaggedTensor):
                 ragged = val
             else:
+                # Expanding / setting last dim of non-list features to be 1D
```

Comment:
is this relevant to `ProcessList`. intuitively `ProcessList` sounds like it might only be transforming list features

Reply:
That is a good point @oliverholworthy . The `ProcessList` is currently used at [core places](https://github.com/NVIDIA-Merlin/models/search?q=process_list) to ensure the features are in good shape for models. When the change making dataloader outputs scalars as 1D happens, we will also need this fix that makes scalars 2D (batch size, 1) for models.
What if we rename `ProcessList` to `PrepareFeatures` and have it as a generic block that works as a translation layer between dataloader and models (used in the same [places](https://github.com/NVIDIA-Merlin/models/search?q=process_list) `ProcessList` is currently used)?

Reply:
The rename sounds like a reasonable thing to do and better matches it's purpose. Can be in another PR if preferred

Reply:
@oliverholworthy I have created another separate PR just for renaming `ProcessList` to `PrepareFeatures`: PR #992